test/NinjaTools.FluentMockServer.Tests/FluentAPI/Builders/FluentExpectationBuilderTests.cs

```diff
@@ -1,82 +1,69 @@
-using System;
 using System.Linq;
 using System.Net;
 using System.Net.Http;
 using System.Threading.Tasks;
 using FluentAssertions;
-using Newtonsoft.Json;
-using NinjaTools.FluentMockServer.Extensions;
-using NinjaTools.FluentMockServer.FluentAPI;
+using Moq;
 using NinjaTools.FluentMockServer.FluentAPI.Builders;
+using NinjaTools.FluentMockServer.Models;
 using NinjaTools.FluentMockServer.Models.ValueTypes;
+using NinjaTools.FluentMockServer.Serialization;
+using NinjaTools.FluentMockServer.Tests.TestHelpers;
 using NinjaTools.FluentMockServer.Tests.TestHelpers.Mocks;
+using NinjaTools.FluentMockServer.Utils;
 using Xunit;
 using Xunit.Abstractions;
+using Times = NinjaTools.FluentMockServer.Models.ValueTypes.Times;
 
 namespace NinjaTools.FluentMockServer.Tests.FluentAPI.Builders
 {
-    public class FluentExpectationBuilderTests
+    public class FluentExpectationBuilderTests : XUnitTestBase<FluentExpectationBuilderTests>
     {
-        private readonly ITestOutputHelper _outputHelper;
-        public FluentExpectationBuilderTests(ITestOutputHelper outputHelper) { _outputHelper = outputHelper; }
-        
-        
-        [Fact]
-        public void Should_Set_Times()
-        {
-            // Arrange
-            Action<IFluentExpectationBuilder> fac = expectationBuilder => expectationBuilder
-                        .OnHandling(HttpMethod.Post, request => request.WithPath("/"))
-                        .RespondOnce(HttpStatusCode.Created, resp => resp.WithDelay(1, TimeUnit.Milliseconds));
-            
-            
-            var setup = new MockServerSetup();
-            var builder = new FluentExpectationBuilder(setup);
-            
-            // Act
-            fac(builder);
-            var expectation = builder.Setup().Expectations.First();
-            var result = expectation.AsJson(); 
-            
-            // Assert
-            _outputHelper.WriteLine(result);
-            result.Should()
-                    .MatchRegex(@"(?m)\s*""times"":\s*\{\s*""remainingTimes"":\s*1,\s*""unlimited"":\s*false\s*}");
-        }
-        
-        [Fact]
-        public void Should_Set_BaseUrl()
+        /// <inheritdoc />
+        public FluentExpectationBuilderTests(ITestOutputHelper output) : base(output)
         {
-            // Arrange
-            const string baseUrl = "http://example.com";
-            var builder = new FluentExpectationBuilder();
-            builder.WithBaseUrl(baseUrl);
-            
-            // Act
-            var setup = builder.Setup();
-            
-            // Assert
-            _outputHelper.WriteLine(JsonConvert.SerializeObject(setup, Formatting.Indented));
-            setup.BaseUrl.Should().Be(baseUrl);
         }
-        
+        // [Fact]
+        // public void Should_Set_Times()
+        // {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4885942)